monadic-html/src/main/scala/mhtml/mount.scala

```diff
@@ -94,15 +100,18 @@ object mount {
 
     def setMetadata(m: MetaData, v: Any, config: MountSettings): Unit = {
       val htmlNode = node.asInstanceOf[dom.html.Html]
-      def set(key: String): Unit =
-        if (v == null) htmlNode.removeAttribute(key)
-        else {
+      def set(key: String): Unit = v match {
+        case null | None | false => htmlNode.removeAttribute(key)
+        case _ =>
           config.inspectAttributeKey(key)
-          // This should be a `config.transformAttributeValue`, but scala-xml does not compile non String attribues...
-          val value = v.toString
+          val value = v match {
+            case Some(x) => x.toString
```

Comment:
This case should never append now that you've added the `case Some` in the other patmat